deployment/permissions/service.userentry.ini

```diff
@@ -0,0 +1,51 @@
+[action_permission_items]
+permissionItem1.service = userentry
+permissionItem1.action = add
+permissionItem1.partnerId = 0
+permissionItem1.param3 =
+permissionItem1.param4 =
+permissionItem1.param5 =
+permissionItem1.tags =
+permissionItem1.permissions = BASE_USER_SESSION_PERMISSION,WIDGET_SESSION_PERMISSION
+
+permissionItem2.service = userentry
+permissionItem2.action = delete
+permissionItem2.partnerId = 0
+permissionItem2.param3 =
+permissionItem2.param4 =
+permissionItem2.param5 =
+permissionItem2.tags =
+permissionItem2.permissions = BASE_USER_SESSION_PERMISSION
+
+permissionItem3.service = userentry
```

Comment:
If you choose not to implement get action, then you don't need to create permission for it.